### DISCUSSION TITLE
Show current IP config on systray tooltip

### DISCUSSION THREAD

**Discussion by Otiel** [snappysnappydog/Simple-IP-Config#114] [Ideas]
Could you show the current IP configuration on a tooltip on the systray icon?

If it matches a Simple-IP-Config profile, could it show the name of the profile on the tooltip as well?

**Comment by KurtisLiggett** [comment]
I could certainly show the current IP on the tooltip.

I'm not sure about showing the profile name. What about a situation where you have 2 profiles with the same configuration? Would you just show the first one it finds in the list?

**Comment by Otiel** [comment]
I hadn't thought of that. It's true that showing the profile name is not required as much as the IP configuration, I would gladly settle with only the IP config shown.

If you still want to go with it, maybe show the profile name only if there is exactly one profile matching the configuration?

**Comment by controlzone** [comment]
I have a few profiles that use the same IP but the gateway or DNS changes. Maybe an option to show profile and/or IP?
